bindings/python/pymongocrypt/version.py

```diff
@@ -12,4 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '1.0.2.dev0'
+from distutils.version import LooseVersion as Version
+
+__version__ = '1.1.0.dev0'
+__min_libmongocrypt_version__ = '1.1.0beta1'
```

Comment:
I think this would be more readable without the dunder naming for `min_libmongocrypt_version`.